sdk/src/main/java/com/mapbox/maps/renderer/RenderHandlerThread.kt

```diff
@@ -46,12 +48,11 @@ internal class RenderHandlerThread {
     }
   }
 
-  fun clearMessageQueue(clearAll: Boolean = true) {
-    if (clearAll) {
-      handler?.removeCallbacksAndMessages(null)
-    } else {
-      handler?.removeCallbacksAndMessages(EventType.SDK)
-    }
+  /**
+   * Clears all messages except of [EventType.DESTROY_RENDERER] messages.
```

Comment:
perhaps add few more words here clarifying why we never actually need `DESTROY_RENDERER` be cleared?

Reply:
Added